server/models/migrations/next/migrate.sql

```diff
@@ -1,11 +1,35 @@
-/**
- * @author: lomamech
- * @description: update migrate.sql file for
- * @date: 2022-01-24
-*/
-UPDATE report SET `report_key` = 'analysis_auxiliary_cashbox', `title_key` = 'REPORT.ANALYSIS_AUX_CASHBOX.TITLE'
```

Comment:
It was not obvious why these got deleted.

Reply:
Probably a failed merge.  Let me fix it.